tests/integrations/mcp/cassettes/mcp_test/test_mcp_client.yaml

```diff
@@ -0,0 +1,415 @@
+interactions:
+- request:
+    body: ''
+    headers:
+      accept:
+      - '*/*'
+      accept-encoding:
+      - gzip, deflate, zstd
+      connection:
+      - keep-alive
+      host:
+      - raw.githubusercontent.com
+      user-agent:
+      - python-httpx/0.28.1
+    method: GET
+    uri: https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window.json
```

Comment:
This block `https://raw.githubusercontent.com/BerriAI/litellm/main/model_prices_and_context_window` can safely be removed from all cassettes (I believe). It fails gracefully and is unrelated to this test

Reply:
removing that block errors out unfortunately.